packages/perseus/src/types.ts

```diff
@@ -194,6 +194,11 @@ export type APIOptions = Readonly<{
      * answered correctly and should no longer be interactive.
      */
     readOnly?: boolean;
+    /**
+     * A boolean that indicates whether the editor interface should be
+     * disabled, preventing content creators from making changes.
```

Comment:
Can you add to this comment explaining an example for when this could happen (i.e. backfills). Just in case someone sees this in the future and wonders "why would we do that?"